static/.htaccess

```diff
@@ -8,6 +8,9 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
+# Identity Authorizations
+RewriteRule ^docs/next/self-managed/identity/user-guide/groups/creating-authorizations-for-a-group/(.*)$ /docs/next/self-managed/identity/user-guide/authorizations/managing-resource-authorizations/ [R=301,L]
```

Comment:
Just to call out for clarity -- this rule will catch the original page, _or any of its children_, and redirect those to the new page. I think that's fine here, because there are no children of this page, but I want to make sure.

Reply:
Thank you for the quick reviews!